POSTINSTALL.md

```diff
@@ -20,7 +20,7 @@ This extension only sends the documents that have been changed -- it does not in
 
 The import script can read all existing documents in a Cloud Firestore collection and insert them into your Algolia `${param:ALGOLIA_INDEX_NAME}` index.
 
-The `npm run import` is for use with the official Firebase Extension [**Search with Algolia**](https://github.com/algolia/algolia-firebase-extension).
+The `npm run import` is for use with the official Firebase Extension [**Search with Algolia**](https://github.com/algolia/firestore-algolia-search).
```

Comment:
I think you can omit this line, since the Algolia Extension context is already clear.